src/angular-business/notification/notification-container/notification-container.component.ts

```diff
@@ -0,0 +1,216 @@
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import {
+  BasePortalOutlet,
+  CdkPortalOutlet,
+  ComponentPortal,
+  DomPortal,
+  TemplatePortal,
+} from '@angular/cdk/portal';
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ComponentRef,
+  ElementRef,
+  EmbeddedViewRef,
+  HostBinding,
+  NgZone,
+  OnDestroy,
+  TemplateRef,
+  ViewChild,
+  ViewEncapsulation,
+} from '@angular/core';
+import { Observable, ReplaySubject, Subject } from 'rxjs';
+
+import { NotificationConfig } from '../notification/notification-config';
+
+export enum NotificationType {
+  SUCCESS = 'success',
+  ERROR = 'error',
+  INFO = 'info',
+  WARN = 'warn',
+}
+
+@Component({
+  selector: 'sbb-notification-container',
+  templateUrl: './notification-container.component.html',
+  styleUrls: ['./notification-container.component.scss'],
+  // In Ivy embedded views will be change detected from their declaration place, rather than
+  // where they were stamped out. This means that we can't have the notification container be OnPush,
+  // because it might cause notifications that were opened from a template not to be out of date.
+  // tslint:disable-next-line:validate-decorators
+  changeDetection: ChangeDetectionStrategy.Default,
+  encapsulation: ViewEncapsulation.None,
+})
+export class NotificationContainerComponent extends BasePortalOutlet implements OnDestroy {
+  private _destroyed = false;
+  /** @docs-private */
+  @HostBinding('class.sbb-notification')
```

Comment:
The lint check does fail if I do this

> Use @HostBinding or
> @HostListener rather than the `host` metadata property (https://angular.io/styleguide#style-06-03)

Or did I understand your comment wrong?

Reply:
This has been changed on master, but is not yet done on this branch